src/systems/renderer.js

```diff
@@ -16,6 +16,8 @@ module.exports.System = registerSystem('renderer', {
     maxCanvasWidth: {default: 1920},
     maxCanvasHeight: {default: 1920},
     physicallyCorrectLights: {default: false},
+    exposure: {default: 1, if: {type: ['ACESFilmic', 'Linear', 'Reinhard', 'Cineon']}},
```

Comment:
Can name values with lowercase if not an acronym for consistency: `linear`, `reinhard`, `cineon`

Can capitalize first letter below:

`string.charAt(0).toUpperCase() + string.slice(1);`

Reply:
"1" is the THREE.js default of no change. 

https://threejs.org/docs/#api/en/renderers/WebGLRenderer.toneMappingExposure

Reply:
Probably can replace 1 with something more user friendly and translate internally.

Reply:
Maybe `no` as `toneMapping` for consistency?

Reply:
Well it's a float it should be set to something like 0.5, or 1.1

Reply:
Like "intensity" on the light component

Reply:
Ah I see that the list is an `if` not a `oneOf` Misread. Sorry

Reply:
This should be:


toneMapping instead of type
```
    exposure: {default: 1, if: {toneMapping: ['ACESFilmic', 'Linear', 'Reinhard', 'Cineon']}},
```

Reply:
Thank you! Great spot!